src/app/features/issue/providers/gitlab/gitlab-api/gitlab-api.service.ts

```diff
@@ -135,21 +140,65 @@ export class GitlabApiService {
   getProjectIssues$(pageNumber: number, cfg: GitlabCfg): Observable<GitlabIssue[]> {
     return this._sendRequest$(
       {
-        url: `${this.apiLink(
+        url: `${this._apiLink(
           cfg,
-        )}/issues?state=opened&order_by=updated_at&per_page=100&page=${pageNumber}`,
+        )}/issues?state=opened&order_by=updated_at&per_page=100&scope=${
+          cfg.scope
+        }&page=${pageNumber}`,
       },
       cfg,
     ).pipe(
       take(1),
       map((issues: GitlabOriginalIssue[]) => {
-        return issues ? issues.map(mapGitlabIssue) : [];
+        return issues ? issues.map((issue) => mapGitlabIssue(issue, cfg)) : [];
       }),
     );
   }
 
+  getFullIssueRef$(issue: string | number, projectConfig: GitlabCfg): string {
```

Comment:
The functionName$ $ is reserved for methods returning an observable. Also the api service is intended to be used for api requests. getProject$, getFullIssueRef look more like utility/mapping functions to me and I would rather put them into a separate file. 

EDIT: I figured that you're just following the pattern established by apiLink. I leave it to you, to decide what pattern you prefer :)